l10n/it-IT.xml

```diff
@@ -38,101 +38,101 @@
+                Value="Questa applicazione non usa lo schermo e funziona in background."/>
 <localized_string Key="AMS_MGR_INTRUPT"
-                Value="Interrupt"/>
+                Value="Interrompi"/>
 <localized_string Key="AMS_MGR_INTRUPT_QUE"
-                Value="How often should %1 ask for permission to interrupt? The interrupted application will exit."/>
+                Value="Quanto spesso dovrebbe %1 chiedere il permesso di interrompersi? L'applicazione interrotta verrà chiusa."/>
 <localized_string Key="AMS_MGR_INTRUPT_QUE_DONT"
-                Value="Don't interrupt and don't ask"/>
+                Value="Non interrompere e non chiedere più"/>
 
 <localized_string Key="PERMISSION_MUTUALLY_EXCLUSIVE_ERROR_MESSAGE"
-                Value="%1 can't be set to not ask because it is not secure when %2 is set to not ask."/>
+                Value="%1 non può essere impostato per non chiedere poiché non è sicuro quando %2 è impostato su non chiedere."/>
 <localized_string Key="PERMISSION_SECURITY_WARNING_ERROR_MESSAGE"
-                Value="It is not secure when %1 and %2 are allowed simultaneously. Proceed anyway?"/>
+                Value="Non è sicuro quando %1 e %2 sono permessi simultaneamente. Procedere comunque?"/>
 <localized_string Key="PERMISSION_AUTO_START_DIALOG_TITLE"
-                Value="Is it OK to automatically start the application?"/>
+                Value="Va bene avviate l'applicazione automaticamente?"/>
 <localized_string Key="PERMISSION_AUTO_START_QUE"
-                Value="%1 wants to register itself to be automatically started.&#010;&#010;Is it OK to register to be automatically started?"/>
+                Value="%1 vuole registrarsi da sola per essere avviata automaticamente.&#010;&#010;Va bene registrarla per essere avviata automaticamente?"/>
 <localized_string Key="AMS_MIDLETSUITEIMPL_CURR_APP"
-                Value="The current application"/>
+                Value="L'attuale applicazione"/>
 <localized_string Key="AMS_MIDLETSUITEIMPL_INTRPT_QUE"
-                Value="Can %1 Interrupt?"/>
+                Value="Può %1 interrompersi?"/>
 <localized_string Key="AMS_MIDLETSUITEIMPL_PUSH_INTERRUPT_DIALOG_TITLE"
-                Value="Is it OK to Automatically Start?"/>
+                Value="Va bene avviare automaticamente?"/>
 <localized_string Key="AMS_MIDLETSUITEIMPL_PUSH_INTERRUPT_QUESTION"
-                Value="Information is arriving for %1. Is it OK for %1 to start up and receive this information? The currently running application will exit.&#010;&#010;Is it OK to start up?"/>
+                Value="Le informazioni stanno arrivando a %1. Va bene che %1 si avvii e ricevi queste informazioni? L'applicazione attualmente avviata si chiuderà.&#010;&#010;Va bene avviarla?"/>
 <localized_string Key="AMS_MIDLETSUITELDR_MIDLETSUITE_NOTFOUND"
-                Value="The MIDlet suite was not found."/>
+                Value="La suite di MIDlet non è stata trovata."/>
```

Comment:
I've found a couple of Italian references that just call them "MIDlet suite" (for example http://www.mobilab.unina.it/Resources/MasterEMCNS-MobileSystems/corsoMS-ParteIII-Lez2.pdf or http://www.di.unipi.it/~f.martel/labj2me06-07/Lezione3.pdf), so that seems the best option.